Dockerfile

```diff
@@ -1,13 +1,13 @@
 FROM node:20.9-slim AS base
-ENV APPDIR /usr/open-org-chart
+ENV APPDIR=/usr/open-org-chart
```

Comment:
I received a log warning from docker that `ENV KEY=VALUE` is legacy and that we should use `ENV KEY VALUE`

Reply:
I received the exact opposite warning

![{A6B69A1E-237A-445F-BD76-708F12E8C9D8}](https://github.com/user-attachments/assets/4284a1c6-818a-41bf-ad5c-a41b82d7ec62)

Reply:
OH! that's it then :) Thanks!

Reply:
I misinterpreted the message, you're correct!